src/Response/JsonResponse.php

```diff
@@ -41,7 +42,7 @@ class JsonResponse extends Response
      * @param int $encodingOptions JSON encoding options to use.
      * @throws InvalidArgumentException if unable to encode the $data to JSON.
      */
-    public function __construct($data, $status = 200, array $headers = [], $encodingOptions = 15)
+    public function __construct($data, $status = 200, array $headers = [], $encodingOptions = 79)
```

Comment:
That last hardcoded number bothers me a lot. Can it be moved to a class constant?